docs/overview.rst

```diff
@@ -161,58 +164,54 @@ FAQ
-read :ref:`the comparison table <comparison>`.
+To see how Kinto compares to CouchDB & Remote Storage, read :ref:`the comparison table <comparison>`.
 
 Can I encrypt my data?
 ----------------------
 
-Kinto server stores any data you pass to it, be it encrypted or not.
-We make it easy to use encryption in our Kinto.js client
-`using transformers <http://kintojs.readthedocs.org/en/latest/api/#transformers>`_.
+Kinto server stores any data you pass to it, whether it's encrypted or not. We believe
+encryption should always be done on the client-side, and we make it easy to use encryption in
+our Kinto.js client `using transformers <http://kintojs.readthedocs.org/en/latest/api/#transformers>`_.
 
 Is there a package for my Operating System?
 -------------------------------------------
 
-No, but it's a great idea. Packaging is hard and we're a small team, so if
-you'd like to help us out by maintaining packages for your favourite OS,
-we'd be delighted to collaborate with you!
+No, but it's a great idea. Maintaining packages for several platforms is time-consuming
+and we're a small team. At this time we're just making sure it's easy to run our server
+using our Makefile or our Dockerfile.
 
-That said, Kinto is :ref:`easy to install with pip <installation>` and
+Kinto is :ref:`easy to install with pip <installation>` and
 we've got `an image set up <https://hub.docker.com/r/kinto/kinto-server/>`_
 on the Docker hub, too.
 
+But if you'd like to help us out by maintaining packages for your favourite OS,
+we'd be delighted to collaborate with you!
+
+
 Why did you chose to use Python rather than X?
 ----------------------------------------------
 
-We know and love `Python <python.org>`_ for its simplicity and short
-learning curve, so it was an obvious choice for the development team. In
-addition, the Operations team at Mozilla is comfortable with deploying and
+We love `Python <python.org>`_ because it's a concise & expressive
+language with powerful data structures & easy to learn,
+so it was an obvious choice for the development team.
+
+In addition, the Operations team at Mozilla is comfortable with deploying and
 managing Python applications in production.
 
-However, the protocol and concepts behind Kinto don't rely on Python *per
-se*, so it is possible to have other Kinto implementations using other
-languages.
+However, Python is just an implementation detail *per se*. Kinto is
+defined by an HTTP protocol that could be implemented in any language.
+
 
 Is it Web Scale?
 ----------------
 
 YES™. Have a look at the ``/dev/null`` backend. ;-)
 
+
 Can I store files inside Kinto?
 -------------------------------
 
-No. Kinto is a JSON storage service and is not designed to store arbitrary
-files. We'd be open to exploring file storage should a solid use-case
-present itself in the future; however, at this time, it's not on our
-roadmap.
+Not yet, but we're working on it and should have it sometimes in 2016.
```

Comment:
We can add a link to https://github.com/Kinto/kinto-attachment/ and mention that is already usable but not stabilized ? What do you think ?